src/tools/illink/external/corert/src/System.Private.CoreLib/shared/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -0,0 +1,156 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is all this corert code needed?  What's the plan for getting rid of it?  Why aren't the current version of these files for coreclr, mono, or nativeaot sufficient?

Reply:
Agreed that we should try to use common reflection sources for this.  cc @dotnet/area-system-reflection

Reply:
Opened https://github.com/dotnet/runtime/issues/77868 to track this work